pkgs/servers/misc/navidrome/default.nix

```diff
@@ -1,20 +1,26 @@
-{ lib, stdenv, fetchurl, ffmpeg, ffmpegSupport ? true, makeWrapper, nixosTests }:
+{ lib, stdenv, pkgs, fetchurl, ffmpeg, ffmpegSupport ? true, makeWrapper, nixosTests }:
```

Comment:
Why are we not using system as an input or even use lib?

Reply:
The derivation works even without importing `system`, I can simply access it, I don't know the reason, I'm trying to learn.
Why `lib` is not ok?

Would be correct this?
```nix
{ stdenv, lib, fetchurl, ffmpeg, ffmpegSupport ? true, makeWrapper, nixosTests }:
```